### DISCUSSION TITLE
Welcome to SubwaySurfersDecrypted Discussions!

### DISCUSSION THREAD

**Discussion by Noobgamer0111** [Noobgamer0111/SubwaySurfersDecrypted#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Samalth** [comment]
Hello, I was just messing with the game files of subway when I actually found out that the old non-encrypted files still work to cheat in the game. If you open an old season hunt folder you can just change the date and the season number to get every important item from the hunt. You could also do this with the wallet file (you don't need the season number for this) except for season tokens which work a little different.

**Reply by Samalth** [reply]
![Screenshot_20230314_223609_File Manager +](https://user-images.githubusercontent.com/127760279/225144288-982bea98-2090-43b7-8c63-efddf6128093.jpg)
![Screenshot_20230314_223630_File Manager +](https://user-images.githubusercontent.com/127760279/225144299-f1fbc3e5-d735-4376-b701-8b859f3c3bdd.jpg)

**Reply by Noobgamer0111** [reply]
While it is known that replacing the old (unencrypted) *.json will work, the main focus is on performing decryption of existing *.json's to be edited.

I'm not saying that your method is invalid, most people don't want to regrind/re-purchase everything back to their current items (like me lol).

A few questions for you:

1. Do the files get encrypted once loaded in to the application? I.e. once you load into the app, and make any kind of purchase/change, does the file get encrypted once you exit the app?
2. Could you please get me more original *.json's? I need to get a copy of every single original *.json with default settings as a control.
3. For (2), could you please add them via a pull request?

**Comment by imcr1** [comment]
Hi , I may be able to code an app to decrypt and make the json files ease to deal with for humans (Non-coders) , you may contact me when you find a way to decrypt 🤩

**Comment by byLeonDE** [comment]
Hello i was working on this topic for a week now and i also messed etc. With the game files and i also tried to find out more about the files and decrypting etc. And i wish we could reach your goal. I dont have a solution but i still have a little thing i found that i need to show you i hope it will help a little. I found it in game files, tower, client, manifest. 
![Screenshot_20230413_224115_cn trinea android developertools](https://user-images.githubusercontent.com/128378414/232062203-46017e82-e98e-4288-ab6c-9c275794e7c4.jpg)

**Reply by byLeonDE** [reply]
If you click the three links there will pop up a website where is only written OK but the main thing is actually the "secret" number

**Reply by byLeonDE** [reply]
Ive researched everywhere but i did not found anything to that i looked in every file that i could look at

**Reply by imcr1** [reply]
Is by any chance that the secret nber is the key for the decryption or maybe it is just an identificator for user-server side of purchases

**Reply by byLeonDE** [reply]
I am not sure

**Reply by Noobgamer0111** [reply]
@byLeonDE Please make this a separate discussion thread. Don't add more to this one.
